.github/workflows/ci.yml

```diff
@@ -283,7 +283,7 @@ jobs:
         run: pytest modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py
       - shell: bash -l {0}
         run: bash <(curl -s https://codecov.io/bash)
-  
+
```

Comment:
Please remove extra newline

Reply:
Okay, will be removed.

Reply:
Actually, we have everywhere blank line between jobs. I think, here blank should be added too.